test/tests/preconditioners/fdp/tests

```diff
@@ -1,16 +1,25 @@
 [Tests]
-  [./test]
-    type = 'Exodiff'
-    input = 'fdp_test.i'
-    exodiff = 'out.e'
-    max_parallel = 1
+  [./jacobian_fdp_coloring_full_test]
+    type = AnalyzeJacobian
+    input = fdp_test.i
+    expect_out = '\nNo errors detected. :-\)\n'
+    recover = false
+    mesh_mode = REPLICATED
+    cli_args = 'Preconditioning/FDP/full=true'
   [../]
-  [./test_standard]
-    type = 'Exodiff'
-    input = 'fdp_test.i'
-    exodiff = 'out.e'
-    max_parallel = 1
-    prereq = 'test'
-    cli_args = "Preconditioning/FDP/finite_difference_type='standard'"
+  [./jacobian_fdp_standard_test]
+    type = AnalyzeJacobian
+    input = fdp_test.i
+    expect_out = '\nNo errors detected. :-\)\n'
+    recover = false
+    mesh_mode = REPLICATED
+    cli_args = 'Preconditioning/FDP/finite_difference_type=standard'
+  [../]
+  [./jacobian_fdp_coloring_diagonal_test_fail]
+    type = AnalyzeJacobian
+    input = fdp_test.i
```

Comment:
You will need to put prereqs in here or these will fail sporadically. The AnalyzeJacobian tester needs to be updated to inherit from `FileTester.py` so that the TestHarness can detect write race conditions. As it stands these tests can run at the same time and will write to the same file so they need to be serialized.